Description: Upstream changes introduced in version 1-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 git-stuff (1-1) unstable; urgency=low
 .
   * Initial release.
 .
 The person named in the Author field signed this changelog entry.
Author: Daniel Baumann <daniel.baumann@progress-technologies.net>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- git-stuff-1.orig/scripts/git-debian-add
+++ git-stuff-1/scripts/git-debian-add
@@ -32,5 +32,7 @@ fi
 
 git add .
 
+chmod 0755 debian/rules
+
 git commit -a -m "Adding debian version ${_VERSION}."
 git-debian-tag ${_TAG} ${_HASH}
